net/nqptp/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+# updated to work with latest source from abrasive
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nqptp
+PKG_VERSION:=1.2.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/mikebrady/nqptp/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=1df1d5edd5b713010d6495b3abca4c1cf4ad8fa6029df0abeb9e4de8e0eb707a
+
+PKG_MAINTAINER:=Ben Klang <bklang@wirehack.net>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING LICENSES nqptp.c
+
+PKG_FIXUP:=autoreconf
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/nqptp
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Not Quite Peer-to-Peer
+  PROVIDES:=nqptp
+  URL:=https://github.com/mikebrady/nqptp
+endef
```

Comment:
This package has no dependencies, I find it weird.

Reply:
It's a very small program. I checked again by looking at the source code and the build instructions, neither list any dependencies. To be safe, I installed this into a fresh OpenWrt SDK with no dependencies installed and it compiled cleanly.

ldd reports:

```
# ldd /usr/bin/nqptp
	/lib/ld-musl-aarch64.so.1 (0x7fa7752000)
	libgcc_s.so.1 => /lib/libgcc_s.so.1 (0x7fa7721000)
	libc.so => /lib/ld-musl-aarch64.so.1 (0x7fa7752000)
```